src/main/resources/config/defaults-storage-driver-kafka.yml

```diff
@@ -7,6 +7,10 @@ storage:
     create:
       key:
         enabled: false
+      headers:
+        Connection: Keep-Alive
```

Comment:
this is HTTP-specific headers which are not suitable for kafka
let's leave the headers empty